comment_help.md

```diff
@@ -8,7 +8,7 @@ description: "Guide on providing feedback"
 
 **Review the documentation.** No account is needed to review the updated version of NIST SP 800-63-3. Simply follow [this link](https://github.com/usnistgov/800-63-3) and enjoy at your leisure. However, if you wish to comment you must…
```

Comment:
For the public comment period, aren't we also accepting comments via more traditional means (spreadsheets emailed to us, etc.)?